understanding/understanding-act-rules.html

```diff
@@ -51,95 +49,72 @@
+                  <h2>テストルールは参考情報である</h2>
                   			
-                  <p><em>Test Rules are informative — that means they are not required for determining conformance.
-                        The basis for determining conformance to WCAG is the success criteria from the WCAG
-                        standard — not the Test Rules.</em></p>
+                  <p><em>テストルールは参考情報である。つまり、適合を判断するために必要ではないことを意味する。WCAG への適合を判断するための基礎は、WCAG テストルールではなく、WCAG 標準の達成基準である。</em></p>
                   			
-                  <p>While W3C's <a href="https://www.w3.org/WAI/standards-guidelines/act/rules/">List of Test Rules for WCAG 2</a> are reviewed by the W3C Accessibility Guidelines Working Group (AGWG), they are not
-                     vetted to the same degree as the W3C Web Content Accessibility Guidelines (WCAG) standard
-                     (called <em>W3C Recommendation</em>). The WCAG standard is the normative reference on determining conformance.
+                  <p>W3C の <a href="https://www.w3.org/WAI/standards-guidelines/act/rules/">List of Test Rules for WCAG 2</a> は、W3C の Accessibility Guidelines Working Group (AGWG) によってレビューされているが、W3C Web Content Accessibility Guidelines (WCAG) 標準 (<em>W3C 勧告</em>と呼ばれる) と同程度に精査されていない。WCAG 規格は、適合を判断するための規範的な基準である。
                   </p>
                   		
                </section>
                		
                		
                <section id="test-rules-are-partial-checks">
                   			
-                  <h2>Test Rules are Partial Checks</h2>
+                  <h2>テストルールは部分的なチェックである</h2>
                   			
-                  <p>Test Rules typically check specific aspects of WCAG success criteria. For example,
-                     that a table cell has a header rather than the entire WCAG 2.2 success criterion 1.3.1
-                     "Info and Relationships", which applies to many more information structures on a web
-                     page. In fact, this example rule would not even check the validity of the table header,
-                     only if the header exists for a given table cell.
+                  <p>テストルールは通常、WCAG 達成基準の特定の側面をチェックする。例えば、テーブルセルには、WCAG 2.2 の全体の達成基準 1.3.1「情報及び関係性」ではない、ヘッダーがある。これは、ウェブページ上のより多くの情報構造に適用される。実際、このサンプルのルールは、特定のテーブルセルにヘッダーが存在する場合に限り、テーブルヘッダーの有効性をチェックさえしない。
                   </p>
                   			
-                  <p>Test Rules are also technology-specific. For example, the aforementioned table header
-                     example would be specific to HTML, possibly enriched with WAI-ARIA roles and properties,
-                     but not to other formats with tables. WCAG 2.2 success criteria are designed to be
-                     technology-agnostic and applicable to all web technologies.
+                  <p>テストルールはまた、技術固有である。例えば、前述のテーブルヘッダーの例は、HTML に固有であり、WAI-ARIA の役割 (role) 及びプロパティが強化されている可能性があるが、テーブルを使用する他の形式には固有ではない。 WCAG 2.2の達成基準は、技術にとらわれず、すべてのウェブ技術に適用できるように設計されている。
```

Comment:
enriched with WAI-ARIA はARIA「を」強化するのではなく、ARIA「によって」強化されているということかと思います。

```suggestion
                  <p>テストルールはまた、技術固有である。例えば、前述のテーブルヘッダーの例は、HTML に固有であり、WAI-ARIA の役割 (role) 及びプロパティによって強化されている可能性もあるが、テーブルを使用する他の形式には固有ではない。 WCAG 2.2の達成基準は、技術にとらわれず、すべてのウェブ技術に適用できるように設計されている。
```